packages/arb-token-bridge-ui/package.json

```diff
@@ -110,6 +110,7 @@
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-react": "^7.31.10",
     "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-plugin-zustand-rules": "chrstph-dvx/eslint-plugin-zustand-rules#4e9222407d797d6767c577296239ecb4eb7da7d1",
```

Comment:
if the repo's maintainer doesn't accept PRs, we might just fork it under the org account?